lib/puppet/provider/templates/invoke_dsc_resource.ps1.erb

```diff
@@ -7,7 +7,10 @@ if (!(Test-Path($PuppetModulesFolder))) {
 $currentState = Get-DscLocalConfigurationManager
 
 if ($currentState.RefreshMode -ne 'Disabled') {
-  throw "DSC LCM RefreshMode must be set to Disabled for Puppet to execute DSC Resources!"
+  $data = [PSCustomObject]@{
```

Comment:
Are there older versions of PowerShell that don't know how to properly serialize a `Hashtable` to JSON or something?

In PS5, this isn't a problem AFAIK

```powershell
@{ "foo" = "bar"} | ConvertTo-Json -Compress


{"foo":"bar"}
```